tests/acceptance/features/bootstrap/Provisioning.php

```diff
@@ -775,6 +779,17 @@ public function deleteLdapUsersAndGroups():void {
 			);
 			$this->rememberThatGroupIsNotExpectedToExist($group);
 		}
+		if (!$this->skipImportLdif) {
+			$this->ldap->delete(
```

Comment:
Could add a comment line
```suggestion
                        //delete all created ldap users
			$this->ldap->delete(
```